modules/ROOT/pages/vpn-about.adoc

```diff
@@ -0,0 +1,70 @@
+= Anypoint VPN
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The Anypoint VPN feature enables you to create a secure connection to systems within your network that are hosted outside of Anypoint Platform. If required, you can create multiple VPNs to connect to different networks or different regions within a network.
+
+[NOTE]
+The number of VPNs you can create depends on the VPN capacity allocated to your account.
+
+To create and maintain a secure connection with your external network, the Anypoint VPN provides the following features:
+
+* Route-based access via 2-byte and 4-byte Autonomous System Numbers (ASN). This defines the routing policies used to connect to your network through the VPN.
+* Reusable IP addresses.
+* Tunnel configuration.
+
+[NOTE]
+Network Address Translation (NAT) is not supported.
+
+== Supported IPsec Configurations
+
+Anypoint VPN supports the IPsec configurations for encrypting data communications between Anypoint Platform and your network:
+
+* IKE version 1
+* AES 128 and 256-bit encryption
+* SHA and SHA-2 hashing
+* Diffie-Hellman (DH) groups:
+** Phase1 DH groups 2, 14-18, 22, 23, 24
+** Phase2 DH groups 2, 5, 14-18, 22, 23, 24
+
+[NOTE]
+The Anypoint VPN endpoint is the responder, so you should establish and maintain connections by initiating traffic from a remote device.
```

Comment:
delete "you should". Maybe "Because the Anypoint VPN endpoint is the responder, be sure to (or "you must"?) establish and maintain connections by initiating traffic from a remote device." (I don't understand what being a responder has to do with the requirement, but network people do?)